### PR TITLE
VQSR soft-filtering, SM interaction updates, enable AS-VQSR

### PR DESCRIPTION
**Pull request by vladsavelyev**
Getting ready to kick off all perth-neuro samples.

Test:
https://batch.hail.populationgenomics.org.au/batches/5611

### CODE REVIEW

File: batch_seqr_loader/utils.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 
 AR_REPO = 'australia-southeast1-docker.pkg.dev/cpg-common/images'
-GATK_VERSION = '4.2.1.0'
+GATK_VERSION = '4.2.2.0-cpgfix00'
```
Comment (lgruen) [inline batch_seqr_loader/utils.py]: What's that `cpgfix00`? ;)

**Review by lgruen — APPROVED** [review]
This is great!! ⭐ 

When we start this for `perth-neuro`, will it also regenerate the data for `acute-care`?
(Or do we need to selectively invalidate the `acute-care` data after GVCF generation?)

File: batch_seqr_loader/utils.py (hunk 1 above)
Comment (vladsavelyev) [reply batch_seqr_loader/utils.py]: Our fork with a couple of fixes:
https://github.com/populationgenomics/gatk/commits/master

Reported one of them back to GATK, hoping to get back to the official release at some point :) 
https://github.com/broadinstitute/gatk/issues/7483

For now, building this image:
https://github.com/populationgenomics/production-pipelines/tree/initial/dockers/gatk

I'll add a comment to the code!

**Comment by vladsavelyev** [comment]
Thanks :)

Yes, it will include acute-care samples into the joint-call as long as all projects is specified in the command line:
https://github.com/populationgenomics/hail-elasticsearch-pipelines/blob/qc-and-soft-filtering/Makefile#L58-L59

File: batch_seqr_loader/utils.py (hunk 1 above)
Comment (lgruen) [reply batch_seqr_loader/utils.py]: Oh, nice! Did you also submit a PR to the GATK repo?

**Pull request merged**